src/lib/playerPlugins/updatePositions.js

```diff
@@ -82,4 +82,24 @@ function inject(serv,player)
     player.entity.onGround = onGround;
     player.emit("positionChanged");
   }
+
+  function setPosition(pos, opt) {
+    opt = opt || {};
+    if (pos) player.entity.position = toFixedPosition(pos);
+    if (typeof opt.yaw != 'undefined') player.entity.yaw=opt.yaw;
+    if (typeof opt.pitch != 'undefined') player.entity.pitch=opt.pitch;
+    
+    player._client.write('position', {
```

Comment:
just keep in mind that this won't move the player after he spawned (got to use the teleport packet for that), maybe the name of the function should reflect that (maybe setInitialPosition)